File: apps/rce-service/src/app/templates/cpp/code.cpp
#include <bits/stdc++.h>
using namespace std;

{{USER_CODE}}

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    string line;
    while (getline(cin, line)) {
        if (line.empty()) continue;

        try {
            stringstream ss(line);
            string token;
            vector<string> args;

            while (getline(ss, token, ',')) {
                token.erase(remove_if(token.begin(), token.end(), ::isspace), token.end());
                args.push_back(token);
            }

            auto ans = {{FUNCTION_NAME}};
            cout << ans << "\n";

        } catch (const exception &e) {
            cerr << "Error: " << e.what() << "\n";
        }
    }
    return 0;
}

File: apps/rce-service/src/app/templates/python/code.py
{{USER_CODE}}

if __name__ == "__main__":
    import sys, ast
    for line in sys.stdin:
        line = line.strip()
        if not line:
            continue
        try:
            args = ast.literal_eval(f"({line},)")
            result = {{FUNCTION_NAME}}(*args)
            print(result)
        except Exception as e:
            print("ERROR:", e, file=sys.stderr)
